src/Symfony/Component/Form/ChoiceList/View/ChoiceGroupView.php

```diff
@@ -46,7 +46,7 @@ public function __construct($label, array $choices = array())
     }
 
     /**
-     * {@inheritdoc}
+     * @return ChoiceGroupView[]|ChoiceView[]
      */
     public function getIterator()
```

Comment:
Perhaps the inheritdoc can stay. If I understand correctly how it is supposed to work, `{@inheritdoc}` only imports the (long) description.